### PR TITLE
[14.0][IMP] base_time_parameter: date options, tests, documentation, fix views

### PR DESCRIPTION
**Pull request by norlinhenrik**

### CODE REVIEW

**Comment by norlinhenrik** [comment]
I think pre-commit error is not related to this module

**Comment by pedrobaeza** [comment]
pre-commit problem fixed in #2452

**Review by nimarosa — APPROVED** [review]

**Comment by nimarosa** [comment]
Weird error I think it's the readme generator. Will try again, since all test are passing. 

/ocabot merge patch

**Comment by nimarosa** [comment]
@pedrobaeza do you know why is the merge is failing? Tests are okay, it's something with the readme generator script, might be a temporal error?

File: base_time_parameter/readme/USAGE.rst [hunk 1]
```diff
@@ -1,21 +1,30 @@
...
-    tax_rate = float(company.get_time_parameter('tax_high_rate', tax_date))
+    # Pass no date: using today's date
+    value = model.get_time_parameter("parameter_code_or_name")
+    # Pass a date or datetime
+    value = model.get_time_parameter("parameter_code_or_name", date=datetime.datetime.now()))
+    # Pass the name of a date/datetime field of the record
+    value = record.get_time_parameter("parameter_code_or_name", "date")
+
+Example of implementation in another module
+-------------------------------------------
```
Comment (pedrobaeza) [inline base_time_parameter/readme/USAGE.rst]: You can't use this underlining, as it's used when the fragments are composed. Use instead `~~~~~~`.

**Pull request merged**